src/render.ts

```diff
@@ -302,8 +317,8 @@ export function renderNegligibleTable({ assets }: { assets: AssetDiff[] }) {
     { align: ['l', 'r', 'r', 'r', 'r'] },
   );
 }
-
-export function renderBiggerTable({ assets }: { assets: AssetDiff[] }) {
+// @ts-ignore
+export function renderBiggerTable({ assets, budgetList }: { assets: AssetDiff[], budgetList:{ name: string, budget: number}[] }) {
```

Comment:
**suggestion** to make this as clear as possible, it might be useful to extract the type of `budgetList` into its own type. Maybe something like `BudgetItem` or `AssetBudget` or `AssetSizeBudget`?